env.example

```diff
@@ -233,6 +233,9 @@ XMPP_DOMAIN=meet.jitsi
 # Internal XMPP server
 XMPP_SERVER=xmpp.meet.jitsi
 
+# Internal XMPP server c2s port
+#XMPP_PORT=5222
```

Comment:
Shouldn't prosody also listen on this port? That's what the name suggests.

Reply:
This has been implemented now